posts/static/posts/rating.css

```diff
@@ -0,0 +1,18 @@
+ .rating {
+    margin: 15px auto;
+    width: 100%;
+    max-width: 500px;
+    border: 3px solid #b3b1b1;
+    height: 30px;
+  }
+
+  .rating_score {
+    font-weight: bold;
+  }
+
+  .ratings-block{
+      background: rgba(255, 255, 255, 0.74);
+      border-radius: 0.5rem;
+      padding: 5px;
+    
+  }
```

Comment:
No newline at the end of the file

Reply:
![image](https://user-images.githubusercontent.com/90772515/167486467-1af5b91b-66f7-48ec-8d17-3f8f04240e32.png)
Somehow it can be seen only when you enter the file

Reply:
Done!